Alignment/HIPAlignmentAlgorithm/data/breakup.sh

```diff
@@ -0,0 +1,76 @@
+#!/bin/bash
```

Comment:
I completely agree, but the HipPy instructions right now point to using data/ for storing alignment objects. I can change the instructions and also put breakup.sh somewhere else, maybe test/?